tests/MongoDB.Driver.Tests/Specifications/transactions/TransactionTestRunner.cs

```diff
@@ -74,40 +77,59 @@ private void Run(BsonDocument shared, BsonDocument test)
             //}
 
             JsonDrivenHelper.EnsureAllFieldsAreValid(shared, "_path", "database_name", "collection_name", "data", "tests");
-            JsonDrivenHelper.EnsureAllFieldsAreValid(test, "description", "clientOptions", "sessionOptions", "operations", "expectations", "outcome");
+            JsonDrivenHelper.EnsureAllFieldsAreValid(test, "description", "clientOptions", "failPoint", "sessionOptions", "operations", "expectations", "outcome");
 
             _databaseName = shared["database_name"].AsString;
             _collectionName = shared["collection_name"].AsString;
 
+            KillAllSessions();
             DropCollection();
             CreateCollection();
             InsertData(shared);
 
-            var eventCapturer = new EventCapturer()
-                .Capture<CommandStartedEvent>(e => !__commandsToNotCapture.Contains(e.CommandName));
+            using (ConfigureFailPoint(test))
```

Comment:
ConfigureFailPoint returns null if the test doesn't use a failpoint.

That's fine, using statements are designed to handle null values.